pkg/backup/backup.go

```diff
@@ -346,9 +347,55 @@ func (kb *kubernetesBackupper) BackupWithResolvers(log logrus.FieldLogger,
 	}()
 
 	backedUpGroupResources := map[schema.GroupResource]bool{}
+	backedUpItems := map[string]bool{}
 	totalItems := len(items)
 
+	var wg sync.WaitGroup
+	
+
+	for _, item:= range items {
+		if item.groupResource.String() == "persistentvolumeclaims" {
+			backedUpItems[item.groupResource.String() + item.name] = true
```

Comment:
This is part of core velero code polling - where BIA of CSI Plugin is called 1 by 1

as part of this proposed change I am calling "BackupITem" in an async manner on all PVCs.
Which will help poll them in parallel

This can be further tweaked by batching number of snapshots triggered in one go.